src/extensions/renderer/canvas/index.js

```diff
@@ -63,13 +64,28 @@ function CanvasRenderer( options ){
   container.appendChild( r.data.canvasContainer );
 
   if( (container.getAttribute('style') || '').indexOf(tapHlOff) < 0 ){
-    container.setAttribute( 'style', ( container.getAttribute( 'style' ) || '' ) + tapHlOff );
+    container.style[tapHlOffAttr] = tapHlOffStyle;
+  }
+
+  var styleMap = {
+    '-webkit-user-select': 'none',
+    '-moz-user-select': '-moz-none',
+    'user-select': 'none',
+    '-webkit-tap-highlight-color': 'rgba(0,0,0,0)',
+    'outline-style': 'none',
+  };
+
+  if(is.ms()) {
+    styleMap['-ms-touch-action'] = 'none';
+    styleMap['touch-action'] = 'none';
   }
 
   for( var i = 0; i < CRp.CANVAS_LAYERS; i++ ){
     var canvas = r.data.canvases[ i ] = document.createElement( 'canvas' );  // eslint-disable-line no-undef
     r.data.contexts[ i ] = canvas.getContext( '2d' );
-    canvas.setAttribute( 'style', '-webkit-user-select: none; -moz-user-select: -moz-none; user-select: none; -webkit-tap-highlight-color: rgba(0,0,0,0); outline-style: none;' + ( is.ms() ? ' -ms-touch-action: none; touch-action: none; ' : '' ) );
+    for(const [k,v] of Object.entries(styleMap)) {
```

Comment:
We can't use `Object.entries()`.  We support IE9.  You can use `Object.keys()` instead though.

I'd stay away from `for of`, in case babel doesn't convert it independently of the system supporting iterables.  It's probably better to use `Object.keys().forEach()`

Reply:
I'll check if this runs without warnings, that will take a moment.
Otherwise this is resolved in 4906fe4f5df7cf334a668def4e678138382fd1f3

Reply:
looks good, deploys without styling errors